lib/ansible/modules/network/nxos/nxos_igmp_interface.py

```diff
@@ -322,7 +322,7 @@ def get_igmp_interface(module, interface):
         'ConfiguredStartupQueryInterval': 'startup_query_interval',
         'StartupQueryCount': 'startup_query_count',
         'RobustnessVariable': 'robustness',
-        'QuerierTimeout': 'querier_timeout',
+        'ConfiguredQuerierTimeout': 'querier_timeout',
```

Comment:
@rahushen is `ConfiguredQuerierTimeout` compatible with nxos7k?

Reply:
@trishnaguha ... Yes, it is supported on N7K as well.